libres/lib/enkf/enkf_main.cpp

```diff
@@ -294,149 +281,158 @@ void enkf_main_exit(enkf_main_type * enkf_main) {
+                std, (const enkf_node_type *)vector_iget_const(ensemble, iens));
+        enkf_node_scale(std, 1.0 / vector_get_size(ensemble));
+
+        if (mean != NULL) {
+            enkf_node_scale(std, -1);
+            enkf_node_iaddsqr(std, mean);
+            enkf_node_scale(std, -1);
+        }
 
-static void enkf_main_node_std( const vector_type * ensemble , const enkf_node_type * mean , enkf_node_type * std) {
-  if (vector_get_size( ensemble ) == 0)
-    util_abort("%s: internal error - calculation std of empty list\n",__func__);
-  {
-    int iens;
-    enkf_node_clear( std );
-    for (iens = 0; iens < vector_get_size( ensemble ); iens++)
-      enkf_node_iaddsqr( std , (const enkf_node_type * ) vector_iget_const( ensemble, iens) );
-    enkf_node_scale(std , 1.0 / vector_get_size( ensemble ));
-
-    if (mean != NULL) {
-      enkf_node_scale( std , -1 );
-      enkf_node_iaddsqr( std , mean );
-      enkf_node_scale( std , -1 );
+        enkf_node_sqrt(std);
     }
-
-    enkf_node_sqrt( std );
-  }
 }
 
+void enkf_main_inflate_node(enkf_main_type *enkf_main, enkf_fs_type *src_fs,
+                            enkf_fs_type *target_fs, int report_step,
+                            const char *key, const enkf_node_type *min_std) {
+    int ens_size = enkf_main_get_ensemble_size(enkf_main);
+    vector_type *ensemble = enkf_main_alloc_node_ensemble(
+        enkf_main, src_fs, key, report_step); // Was ANALYZED
+    enkf_node_type *mean =
+        enkf_node_copyc((const enkf_node_type *)vector_iget_const(ensemble, 0));
+    enkf_node_type *std = enkf_node_copyc(mean);
+    int iens;
 
-void enkf_main_inflate_node(enkf_main_type * enkf_main , enkf_fs_type * src_fs , enkf_fs_type * target_fs , int report_step , const char * key , const enkf_node_type * min_std) {
-  int ens_size                              = enkf_main_get_ensemble_size(enkf_main);
-  vector_type * ensemble                    = enkf_main_alloc_node_ensemble( enkf_main , src_fs , key , report_step );  // Was ANALYZED
-  enkf_node_type * mean = enkf_node_copyc( (const enkf_node_type * ) vector_iget_const( ensemble, 0) );
-  enkf_node_type * std  = enkf_node_copyc( mean );
-  int iens;
-
-  /* Shifting away the mean */
-  enkf_main_node_mean( ensemble , mean );
-  enkf_main_node_std( ensemble , mean , std);
-  enkf_node_scale( mean , -1 );
-  for (iens = 0; iens < ens_size; iens++)
-    enkf_node_iadd( (enkf_node_type * ) vector_iget(ensemble,iens), mean );
+    /* Shifting away the mean */
+    enkf_main_node_mean(ensemble, mean);
+    enkf_main_node_std(ensemble, mean, std);
+    enkf_node_scale(mean, -1);
+    for (iens = 0; iens < ens_size; iens++)
+        enkf_node_iadd((enkf_node_type *)vector_iget(ensemble, iens), mean);
 
-  enkf_node_scale( mean , -1 );
+    enkf_node_scale(mean, -1);
 
-  /*****************************************************************/
-  /*
+    /*****************************************************************/
+    /*
```

Comment:
Indentation of comment now looks weird because the indentation of the code has changed.